README.md

```diff
@@ -1,7 +1,10 @@
 # Jellyfin Theater
 
 Jellyfin Theater is a local client that connects to a Jellyfin server. 
-Currently this takes place by contacting https://
+Currently this takes place by contacting http://
```

Comment:
This line still looks strange... Were you wanting to say "Currently this connects using http protocol"?